File: OrchidCore/src/main/java/com/eden/orchid/impl/themes/tags/PageTag.java
package com.eden.orchid.impl.themes.tags;

import com.eden.orchid.api.compilers.TemplateTag;
import com.eden.orchid.api.options.annotations.Description;

import javax.inject.Inject;
import javax.inject.Singleton;

@Singleton
@Description(value = "Render the main page components and page content.", name = "Page")
public final class PageTag extends TemplateTag {

    @Inject
    public PageTag() {
        super("page", Type.Simple, true);
    }

    @Override
    public String[] parameters() {
        return new String[] {};
    }

}

File: OrchidCore/src/main/kotlin/com/eden/orchid/impl/themes/tags/PageTag.kt
package com.eden.orchid.impl.themes.tags

import com.eden.orchid.api.compilers.TemplateTag
import com.eden.orchid.api.options.annotations.Description

@Description(value = "Render the main page components and page content.", name = "Page")
class PageTag : TemplateTag("page", Type.Simple, true) {

    override fun parameters() = emptyArray<String>()

}
